sources/views/contact-form.js

```diff
@@ -0,0 +1,247 @@
+								},
+								{
+									view: "text",
+									label: "Skype",
+									name: "Skype",
+									validate: webix.rules.isNotEmpty,
+									invalidMessage: "Can not be empty"
+								},
+								{
+									view: "text",
+									label: "Phone",
+									name: "Phone",
+									validate: webix.rules.isNotEmpty,
+									invalidMessage: "Can not be empty"
+								},
+								{
+									view: "datepicker",
+									label: "Birthday",
+									name: "Birthday",
+									format: webix.i18n.longDateFormatStr,
+									validate: webix.rules.isNotEmpty,
+									invalidMessage: "Can not be empty"
+								},
+								{
+									cols: [
+										{
+											type: "clean",
+											localId: "contactPhoto",
+											name: "Photo",
+											height: 200,
+											gravity: 2,
+											template: contact => `<div class="user-info-photo">
+											<img src=${contact.Photo || noPhoto} alt="User photo" /></div>`
+										},
+										{
+											rows: [
+												{},
+												{
+													view: "uploader",
+													localId: "photoUploader",
+													label: "Change photo",
+													accept: "image/png, image/jpg, image/jpeg",
+													autosend: false,
+													css: "button--style"
+												},
+												{
+													view: "button",
+													label: "Delete photo",
+													css: "button--style",
+													click: () => this.deletePhoto()
+												}
+											]
+										}
+									]
+								}
+							]
+						}
+					]
+				},
+				{},
+				{
+					cols: [
+						{view: "button", label: "Cancel", css: "button--style", click: () => this.closeForm()},
+						{
+							view: "button",
+							label: "",
+							css: "button--style",
+							localId: "actionsButton",
+							click: () => this.addEditContact()
+						}
+					]
+				}
+			]
+		};
+		return form;
+	}
+
+	init() {
+		this.form = this.$$("contactForm");
+		this.contactPhoto = this.$$("contactPhoto");
+		const fileReader = new FileReader();
+
+		this.$$("photoUploader").attachEvent("onBeforeFileAdd", (data) => {
+			fileReader.readAsDataURL(data.file);
+			fileReader.onload = () => this.contactPhoto.setValues({Photo: fileReader.result});
+		});
+	}
+
+	urlChange() {
+		const state = this.getParam("status");
+		if (state === "edit") {
+			const id = this.getParam("id", true);
+			this.showContactForm(state, id);
+		}
+		else {
+			this.showContactForm(state);
+		}
+	}
+
+	showContactForm(state, id) {
+		if (id && contacts.exists(id)) {
+			state = "Edit";
+			const item = webix.copy(contacts.getItem(id));
+			this.form.setValues(item);
+			this.contactPhoto.setValues({Photo: item.Photo});
+		}
+		else {
+			state = "Add";
+		}
+		this.$$("header").setValues({state});
+		this.$$("actionsButton").setValue(state);
+	}
+
+	closeForm(id) {
+		this.form.clear();
+		this.form.clearValidation();
+		const status = this.getParam("status");
+		if (status === "edit") {
+			this.show("./contact-info");
```

Comment:
Remove this block. The handler for `showChoosenContactInfo` already does this as needed

Reply:
No,  handler showChoosenContactInfo select item when user editing smth, it's don't need to select item in list.

Reply:
https://snippet.webix.com/7i7mf7yy

If an item is already selected, `list.select(sameId)` won't trigger any events. 
Furthermore, calling `someSubview.show("./anotherSubview");` is not the best choice (as far as I can see, you have correctly avoided it everywhere else).

Reply:
I didn't know, thank you. 
Fixed.